test/hotspot/jtreg/runtime/logging/CompressedOopsTest.java

```diff
@@ -24,7 +24,8 @@
 /*
  * @test
  * @bug 8149991
- * @requires vm.bits == 64 & vm.opt.final.UseCompressedOops == true
+ * @requires vm.bits == 64
+ * @requires vm.flagless
```

Comment:
this test explicitly passes `-XX:+UseCompressedOops` to a child JVM, so the condition hasn't been weakened, if anything this change increases the coverage for JVMs where `UseCompressedOops` isn't true by default as now the test will be included in out-of-box runs.

Reply:
Okay. Thinking more about this case I think the old condition was to stop this test from being run when compressed oops was actually disabled, because it was going to enable them anyway. But now vm.flagless takes care of that case.